GVFS/FastFetch/FastFetchVerb.cs

```diff
@@ -237,7 +244,7 @@ private int ExecuteWithExitCode()
                             try
                             {
                                 bool isBranch = this.Commit == null;
-                                prefetcher.Prefetch(commitish, isBranch);
+                                prefetcher.Prefetch(commitish, isBranch, force: this.Force);

```

Comment:
In general, you only need to use named parameters when passing in a literal. No need when the param name and variable name are equally descriptive.